### PR TITLE
🎨 Create Home Page News Section Component #7

### PR DESCRIPTION
**Pull request by mshekerotokiti**
# Description

I have created the news page, with the read more links that direct to the specific blog page

Reference the issue related to this PR as shown below. Every issue has its own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # 7 
## Type of change

:white_check_mark:  New feature (non-breaking change which adds functionality)

## Screenshot (optional)

Screenshot from 2023-10-02 13-28-16

## Checklist

:white_check_mark:  My code follows the style guidelines of this project
 :white_check_mark:  I have performed a self-review of my code
:white_check_mark: I have commented my code, particularly in hard-to-understand areas
  :white_check_mark: My changes generate no new warnings

### CODE REVIEW

**Comment by mshekerotokiti** [comment]
Fixes #7

File: src/css/style.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    color: #5f5c5c;
+    font-size: 17px;
+    margin-bottom: 45px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+/* Styles for section */
+section {
+    margin-top: 140px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use a Use the margin shorthand property  for all sides you are covering.

File: src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+.news article {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 2px solid #b8b7b7;
+    padding: 10px;
+    margin-left: 110px;
+    margin-right: 100px;
+    margin-top: 50px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: same here

File: .husky/prepare-commit-msg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# gitmoji as a commit hook
+if npx -v >&/dev/null
+then
+  exec < /dev/tty
+  npx -c "gitmoji --hook $1 $2"
+else
+  exec < /dev/tty
+  gitmoji --hook $1 $2
+fi
```
Comment (JAPHETHNYARANGA) [inline .husky/prepare-commit-msg:10]: add this folder to gitignore to avoid pushing these changes

File: src/css/style.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    h1 {
+        margin-left: 20px;
+        margin-bottom: 50px;
+    }
+
+    section {
+        margin-top: 50px;
+        margin-left: 20px;
+        margin-right: 20px;
+    }
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: same

File: src/css/style.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+    /* Adjust margins for .news article on smaller screens */
+    .news article {
+        margin-left: 20px;
+        margin-right: 20px;
+        margin-top: 20px;
+    }
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: same

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
Good work, just go through your files again and  remember to attack phone size screenshot.

**Comment by mshekerotokiti** [comment]
> # Description
> I have created the news page, with the read more links that direct to the specific blog page
> 
> Reference the issue related to this PR as shown below. Every issue has its own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).
> 
> Fixes # 7
> 
> ## Type of change
> ✅ New feature (non-breaking change which adds functionality)
> 
> ## Screenshot (optional)
> Screenshot from 2023-10-02 13-28-16
>  
Screenshot from 2023-10-04 15-03-12

> ## Checklist
> ✅ My code follows the style guidelines of this project ✅ I have performed a self-review of my code ✅ I have commented my code, particularly in hard-to-understand areas ✅ My changes generate no new warnings

**Comment by JAPHETHNYARANGA** [comment]
This is good work, the design is nice.Good work. Just closing the pr since the issue is closed

**Pull request closed without merging**